### PR TITLE
Added more specific variables in the theme file to allow for easier customization

### PR DESCRIPTION
**Pull request by ryanindicio**
# Summary of Changes

To help make Bifold more customizable, I tried to move a lot of the text styling and colors used throughout the app to their own individual assignments in the theme file. Please let me know if there is any feedback or ideas on how to improve these changes or make the naming / grouping more intuitive. 

Also included in this PR - updated ContactListItem styling to better reflect the new chat functionality, and added a line to auto run the post-install patches following an npm install. Also, made a small adjustment to the readme so new users know to add the .env file in the app directory. 

# Related Issues

n/a

# Pull Request Checklist

Tick all boxes below to demonstrate that you have completed the respective task. If the item does not apply to your this PR **check it anyway** to make it apparent that there's nothing to do.

- [x] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this);
- [x] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components;
- [x] Updated documentation as needed for changed code and new or modified features;
- [x] Added sufficient [tests](../__tests__/) so that overall code coverage is not reduced.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

Pro Tip 🤓

- Read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!
- Your PR will likely not be reviewed until all the above boxes are checked and all automated tests have passed.

_PR template adapted from the Python attrs project._

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -2,50 +2,42 @@
 /**
  * @format
  */
-import 'react-native-gesture-handler'
```
Comment (amanji) [inline app/index.js]: Are we updating the linter to use double quotes and different spacing?

File: core/App/components/inputs/CheckBoxRow.tsx [hunk 2]
```diff
@@ -38,9 +38,9 @@ const CheckBoxRow: React.FC<Props> = ({ title, accessibilityLabel, testID, check
         onPress={onPress}
       >
         {checked ? (
-          <Icon name={'check-box'} size={36} color={ColorPallet.brand.primary} />
+          <Icon name={'check-box'} size={36} color={Inputs.checkBoxColor.color} />
```
Comment (amanji) [inline core/App/components/inputs/CheckBoxRow.tsx:41]: Minor nit: Do we want to mix case here? I'm ok with just using camelCase altogether

**Review by amanji — COMMENTED** [review]
I think this is a good step to consolidate/clean up the styling. We in fact had a similar concept before moving towards using semantic names where possible. I think this makes sense now given how things can be overridden where necessary. The only thing I would maybe ask to change the quotation marks and spacing that was already in the repo.

File: core/App/components/inputs/CheckBoxRow.tsx (hunk 2 above)
Comment (jleach) [reply core/App/components/inputs/CheckBoxRow.tsx:41]: +1 For not mixing cases. Let's go with what the app primarily uses now.

File: app/index.js (hunk 1 above)
Comment (jleach) [reply app/index.js]: +1 for keeping it consistent with single quotes.

Comment (ryanindicio) [reply app/index.js]: This totally makes sense, I'll update this. Not sure why my vscode did that

File: core/App/components/inputs/CheckBoxRow.tsx (hunk 2 above)
Comment (ryanindicio) [reply core/App/components/inputs/CheckBoxRow.tsx:41]: The existing style objects are using pascal case, did you want me to update those as well? (Buttons, ColorPallet, etc)

Comment (ryanindicio) [reply core/App/components/inputs/CheckBoxRow.tsx:41]: So for instance with Buttons.primary, do we want the B to remain capitalized?

Comment (jleach) [reply core/App/components/inputs/CheckBoxRow.tsx:41]: I think PascalCase for now. Can the changed to anything later so long as its consistent. I don't think any of the Theme mechanics use camelCase so let's avoid that for now as it introduces another inconsistency.

**Comment by jleach** [comment]
@ryanindicio you'll need to update it against the base branch before we do a review otherwise GitHub will just ask for us to re-review.

**Review by jleach — APPROVED** [review]

**Pull request merged**